test/test-network/conf/23-keep-master.network

```diff
@@ -0,0 +1,6 @@
+[Match]
+Name=dummy98
+
+[Network]
+Bond=bond199
+ActiveSlave=true
```

Comment:
did you intend to add KeepMaster=yes to this test file?

Reply:
nvm, the test file was added in the previous commit but with fewer settings - maybe it should be moved to the next commit entirely?

Reply:
Ouch. Moved.